src/metabase/email/messages.clj

```diff
@@ -55,6 +55,37 @@
      :message-type :html
      :message      message-body)))
 
+(defn send-notification-email
+  "Format and Send an email informing the user about changes to objects in the system."
+  [email object-name modifier-name modifier-email dashboards cards pulses revision-message]
+  {:pre [(string? email)
+         (u/is-email? email)
+         (string? object-name)
+         (string? modifier-name)
+         (string? modifier-email)
+         (string? revision-message)]}
+  (let [data-quote   (rand-nth q/quotations)
+        message-body (stencil/render-file "metabase/email/notification"
+                                          {:emailType "notification"
+                                           :modifier_email modifier-email
+                                           :modifier_name modifier-name
+                                           :object_name object-name
+                                           :has_dashboards (seq dashboards)
```

Comment:
Remember that `seq` actually returns the sequence in question if it's not-empty.

```clojure
(seq [:a :b :c]) -> (:a :b :c)
```

In this case it's probably more appropriate to do either
```clojure
(boolean (seq ...))
```

or
```clojure
(not (empty? ...))
```

If you're going to be using it a lot, it might make sense to define an inline function. `complement` should come in handy in this case
```clojure
(let [not-empty? (complement empty?)] ...)